docs/containertools/README.md

```diff
@@ -0,0 +1,108 @@
+for authentication. 
+
+
+## Buildah and Podman relationship
+
+Buildah and Podman are two complementary Open-source projects that are available on
+most Linux platforms and both projects reside at [GitHub.com](https://github.com)
+with Buildah [here](https://github.com/projectatomic/buildah) and
+Podman [here](https://github.com/containers/libpod).  Both Buildah and Podman are
+command line tools that work on OCI images and containers.  The two projects
+differentiate in their specialization.
+
+Buildah specializes in building OCI images.  Buildah's commands replicate all
+of the commands that are found in a Dockerfile. Buildah’s goal is also to
+provide a lower level coreutils interface to build images, allowing people to build
+containers without requiring a Dockerfile.  The intent with Buildah is to allow other
+scripting languages to build container images, without requiring a daemon.
+
+Podman specializes in all of the commands and functions that help you to maintain and modify
+OCI images, such as pulling and tagging.  It also allows you to create, run, and maintain those containers.
+
+A major difference between Podman and Buildah is their concept of a container.  Podman
+allows users to create "traditional containers" where the intent of these containers is
+to be long lived.  While Buildah containers are really just created to allow content
+to be added back to the container image.   An easy way to think of it is the
+`buildah run` command emulates the RUN command in a Dockerfile while the `podman run`
+command emulates the `docker run` command in functionality.  Because of this you
+cannot see Podman containers from within Buildah or vice versa.
+
+In short Buildah is an efficient way to create OCI images  while Podman allows
+you to manage and maintain those images and containers in a production environment using
+familiar container cli commands.
+
+Some of the commands between the projects overlap:
+
+* build
+The `podman build` and `buildah bud` commands have significant overlap as Podman borrows large pieces of the `podman build` implementation from Buildah. 
+
+* run
+The `buildah run` and `podman run` commands are similar but different.  As explained above podman and buildah have a different concept of a container.  An easy way to think of it is the `buildah run` command emulates the RUN command in a Dockerfile while the `podman run` command emulates the `docker run` command in functionality.  Buildah and podman have someone different concepts of containers, because of this so you can not see podman containers from within buildah or vice versa.
+
+* pull, push 
+These commands are basically the same between the two and either could be used
+
+* commit
+Commit works differently because of the differences in `containers`.  You cannot commit a podman container from buildah nor a buildah container from podman.
+
+* tag, rm, rmi, images 
+These commands are basically the same between the two and either could be used
```

Comment:
I would say rm isn't the same - we'd really prefer you avoid using `buildah rm` on Podman containers. It probably won't make things explode, but Podman doesn't really like having its storage removed from under it

Reply:
good point, reworked.